Hai Reveal: huge continuity error, no text walls in B03-B07, fix other minor problems (#7664)
endless-sky/endless-sky: data/hai/hai reveal 5 philosophers.txt

		conversation
			`The Syndicate security captain in the spaceport is very easy to find. She hands you a pad with a secure validation sequence to confirm her briefing comes from the Navy, co-signed by Xilin.`
			`	She appears to have a posse of other captains with her who conveniently enforce a bubble of privacy around you just by taking up space. "So," she says. "The Republic has agreed to accompany the Hai to precisely one world. Combined intelligence indicates that <planet> is the most likely place for any kidnapped Hai to be, assuming they exist. The blockades that have gone up are to secure this area of space against the secret of this exercise... or at least try to.`
			`	"There's going to be a battle, and you have a critical role entrusted to you for coordinating the fleet. There can be no detours. Once we start moving it will be one jump to the rendezvous and then two to our target. Are you ready for it?"`
			choice
				`	"I'm ready now."`
				`	"I need time to prepare."`
					goto prepare

> 			`	"Very good." So far she hasn't introduced herself but as she puts the pad away, her flight jacket moves enough to see 'Captain Valeria Estrada' printed on the jumpsuit underneath. "Now, to bring you up to speed: At least two pirate groups at <planet> have been involved in smuggling or kidnappings. <planet> is a fortress of anonymity and discretion, as well as being a literal fortress. An agreed upon force of overwhelming numbers has been prepared at Mountaintop. The objective is to overwhelm the defenders, prevent any escapes, and occupy the planet while they ferret out all the groups intelligence has flagged.`

Reviewer: ```suggestion
			`	"Very good." So far she hasn't introduced herself, but as she puts the pad away her flight jacket moves enough to for you to see 'Captain Valeria Estrada' printed on the jumpsuit underneath. "Now, to bring you up to speed: At least two pirate groups at <planet> have been involved in smuggling or kidnappings. <planet> is a fortress of anonymity and discretion, as well as being a literal fortress. An agreed upon force of overwhelming numbers has been prepared at Mountaintop. The objective is to overwhelm the defenders, prevent any escapes, and occupy the planet while they ferret out all the groups Intelligence has flagged.`
```

> 			`	"Very good." So far she hasn't introduced herself but as she puts the pad away, her flight jacket moves enough to see 'Captain Valeria Estrada' printed on the jumpsuit underneath. "Now, to bring you up to speed: At least two pirate groups at <planet> have been involved in smuggling or kidnappings. <planet> is a fortress of anonymity and discretion, as well as being a literal fortress. An agreed upon force of overwhelming numbers has been prepared at Mountaintop. The objective is to overwhelm the defenders, prevent any escapes, and occupy the planet while they ferret out all the groups intelligence has flagged.`

Reviewer: ```suggestion
			`	"Very good." So far she hasn't introduced herself, but as she puts the pad away her flight jacket moves enough for you to see 'Captain Valeria Estrada' printed on the jumpsuit underneath. "Now, to bring you up to speed: At least two pirate groups at <planet> have been involved in smuggling or kidnappings. <planet> is a fortress of anonymity and discretion, as well as being a literal fortress. An agreed upon force of overwhelming numbers has been prepared at Mountaintop. The objective is to overwhelm the defenders, prevent any escapes, and occupy the planet while they ferret out all the groups Intelligence has flagged.`
```

Author: "Intelligence" with a capital "I" doesn't make sense here because it implies there is an organization called "Intelligence" that collected it. With a lower-case "i," it means an unspecified government intelligence-gathering organization has flagged it. That is correct usage, albeit an obscure one. I'll rewrite that sentence after I have time to ponder it.

Author: Actually, come to think of it, keeping the original lower-case "i" intelligence makes the most sense because it is a special ops marine speaking. She'd use special ops jargon, so "intelligence" makes the most sense than spelling out the name of the intelligence agency (which she's probably not allowed to do anyway).

Reviewer: There is an organisation called Intelligence under the Republic.

Reviewer: All right. 
```suggestion
			`	"Very good." So far she hasn't introduced herself, but as she puts the pad away her flight jacket moves enough for you to see 'Captain Valeria Estrada' printed on the jumpsuit underneath. "Now, to bring you up to speed: At least two pirate groups at <planet> have been involved in smuggling or kidnappings. <planet> is a fortress of anonymity and discretion, as well as being a literal fortress. An agreed upon force of overwhelming numbers has been prepared at Mountaintop. The objective is to overwhelm the defenders, prevent any escapes, and occupy the planet while they ferret out all the groups intelligence has flagged.`
```

			`	"The Navy, Unfettered, and Hai fleets will travel from Mountaintop to meet you. We must fly with all haste to <waypoints> to meet them, where you will lead the fleet and coordinate the attack. My security force will follow, but until we get to Sumar don't worry about waiting for us. We'll handle ourselves. Once we're all together in Sumar, your job will be to make sure everyone moves as one for each jump from there to <system>."`

> 			`	She lowers her voice even further, despite the privacy afforded already. "There will be two Navy Carrier groups, our full Syndicate Special Security Detachment, the Unfettered Expeditionary Fleet, and a full Hai Taskforce. I haven't heard of a fleet this large in one place since the Alpha Wars. This coordination role is not a joke. There's lots of scope for things to go wrong. We will go in, clean up, and search for Hai. Parliament will reassess the scope of concern later depending on what we find down there. We follow your lead Captain <last>."`

Reviewer: ```suggestion
			`	She lowers her voice even further, despite the privacy afforded already. "There will be two Navy Carrier groups, our full Syndicate Special Security Detachment, the Unfettered Expeditionary Fleet, and a full Hai Taskforce. I haven't heard of a fleet this large in one place since the Alpha Wars. This coordination role is not a joke. There's lots of scope for things to go wrong. We will go in, clean up, and search for Hai. Parliament will reassess the scope of concern later, depending on what we find down there. We follow your lead, Captain <last>."`
```

> 			`	She lowers her voice even further, despite the privacy afforded already. "There will be two Navy Carrier groups, our full Syndicate Special Security Detachment, the Unfettered Expeditionary Fleet, and a full Hai Taskforce. I haven't heard of a fleet this large in one place since the Alpha Wars. This coordination role is not a joke. There's lots of scope for things to go wrong. We will go in, clean up, and search for Hai. Parliament will reassess the scope of concern later, depending on what we find down there. We follow your lead, Captain <last>."`

Reviewer: I've mentioned this already and it's a problem for another PR, but I swear the Navy has more ships here than the entirety of the FW campaign. That has to be too many people involved to keep even a pretense of a secret, even when they were just guarding the wormhole.

Author: That is intentional. Agent Kopkow comments in a later mission that the Navy's cover-up has become a laughingstock and frequent target of memes.

Reviewer: It's still a lot of ships and I think it could be toned down just a little bit.

Author: Have you tried to dominate Earth? This fleet is miniscule compared to the Earth defense fleets. 

In the code, it explains that the vast fleets are for show, and are intended to be preposterously large.

Reviewer: Well, I've never tried to, all I know about Earth's fleet is that it just takes a long time with an Arfecta. Keep it in then.

		has "Hai Reveal [B02] Join the Fleet: done"
	on offer
		conversation

> 			`While you join the fleet that lands to occupy the spaceport, other assault ships pursue priority targets elsewhere on <origin>. This leads to a significant wait until the port is secure before everyone needed can reconvene to discuss the next steps.`

Reviewer: ```suggestion
			`While you join the fleet that lands to occupy the spaceport, other assault ships pursue priority targets elsewhere on <origin>. There's a significant wait to ensure that the port is secure before everyone needed can reconvene to discuss the next steps.`
```
Surely there's enough ships around to occupy Stormhold's main spaceport? There's probably more ships here in Alcyone than fought in the entire Free Worlds war.

Author: The point being made here is that Stormhold is, indeed, a planet of fortresses, as the planet description implies. There are numerous pirates on the ground in well-fortified positions, shooting at anyone who enters the spaceport, so you have to wait for special ops troops to secure the area. Subduing a well-armed and hostile population of millions holed up in bunkers will take wave after wave of troop transports and a prolonged operation. Even one spaceport is a major operation.

Reviewer: Alright, but I still think that my suggestion sounds better in general.

Author: Yes, sorry, I misread it. I see what's bothering me now: it should be "port has been secured" rather than "port is secure." I will do that, and put in your changes too.

Author: I rewrote that part and (I think) I incorporated your suggestions.

			`	"General Choot'k, would you agree with the human's assessment of the effort required here?" asks Teeneep.`
			`	The Unfettered warlord does not even hesitate to respond, "Yes. Even with our superior capacities we will be here a while."`
				goto briefing
			label disinformation
			action
				set "Hai Reveal [B03] Contact Giti: label disinformation"
			`	Batu clears his throat and Danforth glances at him before evidently remembering something discussed earlier. "I'm sorry Captain <last> and Elder Teeneep. I must advise you that at this stage all hope of keeping the Hai a secret is lost. Even if we could convince all factions to keep the secret, the blockade has drawn enough attention so all that remains is to resolve issues before the Hai become irretrievably common knowledge."`
			`	Teeneep responds, "We all knew that day would come soon. Be glad we're here together to usher it in peacefully and with strength."`
				goto briefing
			label squirrels
			action
				set "Hai Reveal [B03] Contact Giti: label squirrels"
			`	"Our place is here, Captain <last>," says Teeneep, definitively.`

> 			`	Danforth looks between them. "Perhaps, Elder, you and the Ambassador would find it more comfortable to return to Mountaintop? There will be little diplomacy here, and scarcely more safety, and it would seem to me that even with Batu here as the deputy minister's eyes on the ground, it would be easier to conduct your responsibilities directly."`

Reviewer: ```suggestion
			`	Danforth looks between them. "Perhaps, Elder, you and the Ambassador would find it more comfortable to return to Mountaintop? There will be little diplomacy here, and scarcely more safety, and it would seem to me that, even with Batu here as the Deputy Minister's eyes on the ground, it would be easier to conduct your responsibilities directly."`
```

				goto briefing

			label offended
			`	Everyone looks surprised or offended, as if you are the court jester who thought he was king. They're dismissive of your comment, as they begin to discuss your departure.`
				goto done
			label honored
			`	The leaders all look pleased. General Choot'k says, "You've surpassed my expectations, Captain <last>. I've never seen a fleet so large, but you led it to victory with the speed and skill of a veteran general." Danforth looks just as impressed.`
			label done
			action

> 			`	Danforth explains, "For now, all communication in and out of the area is blocked at the relay unless its to or from specific high-level Navy or Syndicate sources. Captain <last>, I'm afraid that also includes you. We don't have time or equipment to harden your systems against potential intrusion, but you do have the nearly exclusive freedom to pass through the blockade. You're a pilot, not a ground combat troop operator, so you should pursue other matters."`

Reviewer: ```suggestion
			`	Danforth explains, "For now, all communication in and out of the area is blocked at the relay unless it is to or from specific high-level Navy or Syndicate sources. Captain <last>, I'm afraid that also includes you. We don't have time or equipment to harden your systems against potential intrusion, but you do have the nearly exclusive freedom to pass through the blockade. You're a pilot, not a ground combat troop operator, so you should pursue other matters."`
```

> 			`	On your way out Sayari catches you and suggests meeting aboard Teeneep's ship to discuss some things.`

Reviewer: ```suggestion
			`	On your way out Sayari catches you and suggests meeting aboard Teeneep's ship to discuss the kidnappings.`
```
Anything is better than 'some things'. Maybe there's still a better word than 'kidnapping' though.

Author: They Hai and Captain Last are trying to keep the cadmium poisoning secret (not sure why) so she's being intentionally vague. I'll update this, so she actually says something vague.

Author: Now Sayari actually speaks, and there's a choice that breaks up the conversation, so there's a scene transition.

> 			`	Once there she pulls up information on known or possible smuggling operations used to justify the <origin> operation. Her focus is on what they hadn't reconciled. "Unfortunately, while many matches were found there was nothing in the new data that conclusively linked to the mysterious symbol. No human escapees had been tested for mild cadmium poisoning, so that was a dead end too. I know it's been thrust into the background pretty heavily, but did you find anything yourself? We need to decide if we're setting this aside or not."`

Reviewer: ```suggestion
			`	Once there she pulls up information on known or possible smuggling operations used to justify the <origin> operation. Her focus is on what they hadn't reconciled. "Unfortunately, while many matches were found, there was nothing in the new data that conclusively linked to the mysterious symbol. No human escapees had been tested for mild cadmium poisoning, so that was a dead end too. I know it's been thrust into the background pretty heavily, but did you find anything yourself? We need to decide if we're setting this aside or not."`
```

			choice
				`	"I've met a reseracher named Giti who has been quite informative."`
					goto giti
				`	"I need to leave the communication blackout to check."`

> 			`	You explain that, aside from a short window, you realize that you've been behind a communications blackout for most of the recent events. You'll have to go outside the blockade to follow up on your possible lead.`

Reviewer: Sayari or Teeneep need to respond to you here.

Author: I will revisit that part. My removal of that "action choice" was clumsy.

Author: I've rewritten that text.

					to display
						has "Hai Reveal [B03] Contact Giti: label operation"
						has "Hai Reveal [B03] Contact Giti: label disinformation"
						has "Hai Reveal [B03] Contact Giti: label squirrels"
				`	"Excellent work, everyone! Carry on!"`
					goto offended
					to display
						has "Hai Reveal [B03] Contact Giti: label operation"
						has "Hai Reveal [B03] Contact Giti: label disinformation"
						has "Hai Reveal [B03] Contact Giti: label squirrels"

			label operation
			action
				set "Hai Reveal [B03] Contact Giti: label operation"
			`	Danforth begins the report. "Stage one has been successful. However, it looks like we're going to be here a while. <origin> is a rabbit warren of hidden places. Even with the Hai resources on top of everything we've brought, it could take weeks to root things out sufficiently."`
			`	"General Choot'k, would you agree with the human's assessment of the effort required here?" asks Teeneep.`
			`	The Unfettered warlord does not even hesitate to respond, "Yes. Even with our superior capacities we will be here a while."`
				goto briefing
			label disinformation
			action
				set "Hai Reveal [B03] Contact Giti: label disinformation"

> 			`	Batu clears his throat and Danforth glances at him before evidently remembering something discussed earlier. "I'm sorry Captain <last> and Elder Teeneep. I must advise you that at this stage all hope of keeping the Hai a secret is lost. Even if we could convince all factions to keep the secret, the blockade has drawn enough attention so all that remains is to resolve issues before the Hai become irretrievably common knowledge."`

Reviewer: ```suggestion
			`	Batu clears his throat and Danforth glances at him before evidently remembering something discussed earlier. "I'm sorry, Captain <last> and Elder Teeneep. I must advise you that at this stage all hope of keeping the Hai a secret is lost. Even if we could convince all factions to keep the secret, the blockade has drawn too much attention, so all that remains is to resolve our common issues before the Hai become irretrievably common knowledge."`
```

			`	Teeneep seems to bristle but Sayari smoothly cuts in. "You should not concern yourself with our comfort, Admiral. It would not be seemly for the Elder to abandon the field while another leader remains." She looks pointedly at Choot'k. "You may have forgotten also but there are Hai representatives of two governments in this room. It would also be unseemly for our ships to act in concert with the Unfettered without a diplomatic liaison here. And so I too have my place here. Things have happened quickly, of course, but my colleague Hiyamaharu will soon arrive at Mountaintop to act as my counterpart there. I trust things are being attended to, but you might consider checking that your people are properly aware of that."`
				goto briefing

			label offended
			`	Everyone looks surprised or offended, as if you are the court jester who thought he was king. They're dismissive of your comment, as they begin to discuss your departure.`
				goto done
			label honored

> 			`	The leaders all look pleased. General Choot'k says, "You've surpassed my expectations, Captain <last>. I've never seen a fleet so large, but you led it to victory with the speed and skill of a veteran general." Danforth looks just as impressed.`

Reviewer: ```suggestion
			`	The leaders all look pleased. General Choot'k says, "You've surpassed my expectations, Captain <last>. I've never seen a fleet so large, but you led it to victory with the speed and skill of a veteran admiral." Danforth looks just as impressed.`
```
Should general be admiral, or has this been thought of already?

			`	"So, the notation is actually quite dense, and is mostly some probably meaningless prose about something called 'Elenchus'. It sounds aspirational. However, it also refers to a secret gateway. I'm sending you some coordinates." You receive a simple text message a moment later with some system coordinates in it.`
			choice
				`	"What is Elenchus?"`
				`	"I've never seen this system on a star map. What do you expect to find there?"`
			`	Giti ignores your question and continues. "What I've just send you is the modern representation of coordinates in the notation. Now it's possible there's some kind of code or cipher, in which case they'd be wrong, but if they are direct from the notation, they indicate a system in Syndicate space that is not on the regular star maps. It must require a jump drive to access, but fortunately I know you happen to have one."`

> 			`	She shrugs. "Of course I have no idea how they'd have a jump drive, but that's not my problem. Whether you find something or not come back and see me. If that turns up bust I found an online community who seem to be a support group for people who've had family members kidnapped. The missing person reports for some of them are suspiciously close matches or share features with those in the files you left for me. Not sure if it'll reveal anything but we can look into it if you don't find anything. I haven't looked deeply though, it just came up yesterday while I was chasing down loose threads."`

Reviewer: ```suggestion
			`	She shrugs. "Of course, I have no idea how they'd have a jump drive, but that's not my problem. Whether you find something or not, come back and see me. If that turns up bust, I found an online community who seem to be a support group for people who've had family members kidnapped. The missing person reports for some of them are suspiciously close matches or share features with those in the files you left for me. Not sure if it'll reveal anything, but we can look into it if you don't find anything. I haven't looked deeply though, it just came up yesterday while I was chasing down loose threads."`
```
I wonder if the repetition is genuinely part of her character, so I've kept it.

Author: I reduced the redundancy of that paragraph.

			choice
				`	"How do you know I have a Jump Drive?"`
				`	"Can you tell me more about the victim support group?"`
				`	(Give up on getting any more information out of her.)`
					goto giveup
			`	She seems rushed and ignores you, picking up a bag and putting some items from her desk into it.`
				goto end
			label giveup
			`	She picks up a bag and starts putting some items from her desk into it.`
			label end

> 			`	"In any event, I really must go. Check out those coordinates, and then come see me. Fly safe Captain!"`

Reviewer: ```suggestion
			`	"In any event, I really must go. Check out those coordinates, and then come see me. Fly safe, Captain!"`
```

				`	"I was a little caught up, actually."`
				`	"What do you know about the blockades?"`
			`	As you draw breath to speak she cuts you off. "I actually don't have a lot of time or I'll miss my ride, so we'll skip the pleasantries. Obviously if you're calling me back you want to know what I found about the notation. That too is something you have good timing on, I was delayed on that until just a couple days ago. Now where was that file... ah here it is.`
			`	"So, the notation is actually quite dense, and is mostly some probably meaningless prose about something called 'Elenchus'. It sounds aspirational. However, it also refers to a secret gateway. I'm sending you some coordinates." You receive a simple text message a moment later with some system coordinates in it.`
			choice
				`	"What is Elenchus?"`
				`	"I've never seen this system on a star map. What do you expect to find there?"`

> 			`	Giti ignores your question and continues. "What I've just send you is the modern representation of coordinates in the notation. Now it's possible there's some kind of code or cipher, in which case they'd be wrong, but if they are direct from the notation, they indicate a system in Syndicate space that is not on the regular star maps. It must require a jump drive to access, but fortunately I know you happen to have one."`

Reviewer: ```suggestion
			`	Giti ignores your question and continues. "What I've just sent you is the modern representation of coordinates in the notation. Now it's possible there's some kind of code or cipher, in which case they'd be wrong, but if they are direct from the notation, they indicate a system in Syndicate space that is not on the regular star maps. It must require a jump drive to access, but fortunately I know you happen to have one."`
```

> 			`	She shrugs. "Of course, I have no idea how they'd have a Jump Drive, but that's not my problem. Whether you find something or not, come back and see me. If that turns up bust, I found an online community who seem to be a support group for people who've had family members kidnapped. The missing person reports for some of them are suspiciously close matches or share features with those in the files you left for me. I haven't looked deeply though, it just came up yesterday while I was chasing down loose threads."`

Reviewer: ```suggestion
			`	She shrugs. "Of course, I have no idea how they'd have a jump drive, but that's not my problem. Whether you find something or not, come back and see me. If that turns up bust, I found an online community who seem to be a support group for people who've had family members kidnapped. The missing person reports for some of them are suspiciously close matches or share features with those in the files you left for me. I haven't looked deeply though, it just came up yesterday while I was chasing down loose threads."`
```

			choice

> 				`	"How do you know I have a Jump Drive?"`

Reviewer: ```suggestion
				`	"How do you know I have a jump drive?"`
```
I think this is public knowledge. The player does jump to Sol with one during the Free Worlds campaign.

Author: I have swapped that with the next question, so the default option is not to ask a rhetorical question.

> 		has "Hai Reveal [B04] The Road Not Taken: done"

Reviewer: I can't comment on line 395, but the description isn't in line with other ones. I'd keep Monopoly out of this.

Author: It is removed.

	label talk
	`	The man, whose Republic Intelligence uniform is labeled with the name "Agent Kopkow", nods very firmly and gestures for you to take one of the only seats in the room. He takes the one opposite and Giti sits at her desk. The three agents remain standing.`
	`	You've had some encounters with Republic Intelligence, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle: murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament... which mostly mean it's beholden to the interests of Parliament.`
	choice
		`	"What's the problem, agent?"`
			goto uncomplaining
		`	"I'm on an urgent mission that is critical for three governments. Don't bar my way."`
	`	The agents smile and Kopkow says, "We don't respond well to threats."`
	label uncomplaining

> 	`	"Now Captain, your friend Giti here has tripped over a flag buried in our systems that marks what you're looking into as an item of particular and specific interest to Syndicate Internal Affairs. In fact, it is an interest which they have exclusive sole jurisdiction over, even over and above us." He looks at you meaningfully. "That requires an act of parliament in a closed session, which has not occurred for a significant length of time, if you catch my drift."`

Reviewer: ```suggestion
	`	"Now Captain, your friend Giti here has tripped a flag in our systems that marks what you're looking into as an item of particular interest to Syndicate Internal Affairs. In fact, it is an interest which they have exclusive sole jurisdiction over, even over and above us." He looks at you meaningfully. "That requires an Act of Parliament in a closed session, which has not occurred for a significant length of time, if you catch my drift."`
```

	label talk
	`	The man, whose Republic Intelligence uniform is labeled with the name "Agent Kopkow", nods very firmly and gestures for you to take one of the only seats in the room. He takes the one opposite and Giti sits at her desk. The three agents remain standing.`
	`	You've had some encounters with Republic Intelligence, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle: murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament... which mostly mean it's beholden to the interests of Parliament.`
	choice

> 		`	"What's the problem, agent?"`
> 			goto uncomplaining
> 		`	"I'm on an urgent mission that is critical for three governments. Don't bar my way."`
> 	`	The agents smile and Kopkow says, "We don't respond well to threats."`

Reviewer: ```suggestion
		`	"What's the problem?"`
			goto uncomplaining
		`	"I'm on an urgent mission that is critical for three governments. Don't bar my way."`
	`	Uncannily, the agents all smirk at you at the same time. Kopkow responds flatly, "We don't respond well to threats."`
```
The player acts too casually in a moment where they had the option to be killed just before.

	label uncomplaining

> 	`	"Now Captain, your friend Giti here has tripped a flag in our systems that marks what you're looking into as an item of particular interest to Syndicate Internal Affairs. In fact, it is an interest which they have exclusive sole jurisdiction over, even over and above us." He looks at you meaningfully. "That requires an Act of Parliament in a closed session, which has not occurred for a significant length of time, if you catch my drift."`

Reviewer: ```suggestion
	`	"Now Captain, your friend Giti here has tripped a flag in our systems that marks what you're looking into as an item of particular interest to Syndicate Internal Affairs. In fact, it is an interest which they have exclusive and sole jurisdiction over, even over and above us." He looks at you meaningfully. "That requires an Act of Parliament in a closed session, which has not occurred for a significant length of time, if you catch my drift."`
```
Missed something else just then

> 	`	"Now Captain, your friend Giti here has tripped a flag in our systems that marks what you're looking into as an item of particular interest to Syndicate Internal Affairs. In fact, it is an interest which they have exclusive and sole jurisdiction over, even over and above us." He looks at you meaningfully. "That requires an Act of Parliament in a closed session, which has not occurred for a significant length of time, if you catch my drift."`

Reviewer: ```suggestion
	`	"Now, Captain <last>, your friend Giti here has tripped a flag in our systems that marks what you're looking into as an item of particular interest to Syndicate Internal Affairs. In fact, it is an interest which they have exclusive and sole jurisdiction over, even over and above us." He looks at you meaningfully. "That requires an Act of Parliament in a closed session, which has not occurred for a significant length of time, if you catch my drift."`
```

> 	`	He clears his throat, "While you were making your way here we have accessed your ship's logs with a full warrant to search through your activity since your last contact with Giti. It is all perfectly legal and you'd be wise not to object. Since we have determined that you have found... something, and that it is meaningful, we must send you to Hephaestus. We request that you take Giti, any information or equipment you were planning to take, and go immediately and directly. Do you understand?"`

Reviewer: ```suggestion
	`	He clears his throat. "While you were making your way here, we have accessed your ship's logs with a full warrant to search through your activity since your last contact with Giti. It is all perfectly legal and you'd be wise not to object. Since we have determined that you have found... something, and that it is meaningful, we must send you to Hephaestus. We request that you take Giti, any information or equipment you were planning to take, and go immediately and directly. Do you understand?"`
```
Isn't the Syndicate capital on Foundry, or do they make it hard for themselves by leaving the CEO there?

Author: You're not visiting the CEO. Some lower-level manager who deals with internal security wants to see you.

Author: Accessing the ship's logs can be problematic lore-wise though, as it means anyone could easily discover the Remnant. I will change the explanation of how they got this information.

Reviewer: Nevermind.

	label talk
	`	The man, whose Republic Intelligence uniform is labeled with the name "Agent Kopkow", nods very firmly and gestures for you to take one of the only seats in the room. He takes the one opposite and Giti sits at her desk. The three agents remain standing.`

> 	`	You've had some encounters with Republic Intelligence, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle: murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament... which mostly mean it's beholden to the interests of Parliament.`

Reviewer: ```suggestion
	`	You've had some encounters with the Republic Intelligence Bureau, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle: murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament... which mostly means it's beholden to the interests of the members of Parliament.`
```
Pretty sure they have Bureau at the end of it.

> 	`	You've had some encounters with Republic Intelligence, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle: murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament... which mostly mean it's beholden to the interests of Parliament.`

Author: No. I just checked all the data files, and it never has Bureau at the end.

		not "event: knows about keystones and wormholes"
	on offer
		conversation
			`At length you return again to tell Giti about the system with the unstable wormhole in it. You aren't required to land elsewhere on the planet this time, so it's a shorter, if equally crowded, journey to her office.`
			`	After the both of you spend some time pouring over things for a while, she concludes that they must have some sort of special feature they're using to access it. "It just doesn't make sense that there'd be a system there, that they have coordinates to, that they're not using. Especially when we can't find them anywhere else."`
			choice
				`	"Do you have any other leads I can follow?"`
				`	"Even if I've found them, I don't know what they're doing or how. Any ideas?"`

> 			`	She looks at the many tabs displaying the information you've brought or she's found. "You know, if there's something special needed. When I was searching things up you might remember that I found an online community. I came across a man who might, just maybe, know something useful. He claims to have found something odd related to cases in the set in your data, and one of the cases in your set is the disappearance of his wife which is where his motivation comes from.`

Reviewer: ```suggestion
			`	She looks at the many tabs displaying the information you've brought or she's found. "You remember that online community on unexplained kidnappings? I came across a man who might, just maybe, know something useful. He claims to have found something odd related to cases in the set in your data, and one of the cases in your set is the disappearance of his wife, which is where his motivation comes from.`
```

			`	"His theory is... well it's a bit crackpot for the local authorities. I wouldn't have believed it either were it not for the vastly superior swathe of information we now have at our disposal, but it might be what you need. I think you should go seek him out on Tarazed."`

> 			`	She rummages around and then presents you with a data stick. "This has the information he gave me to follow him up. I convinced him to trust me so hopefully he's receptive to you. It also has all the data that explicitly identifies all the ship ID's that are associated with our suspicious cases. That's the best I've got for you though. If this doesn't pan out we'll need to find different assistance."`

Reviewer: ```suggestion
			`	She rummages around and then presents you with a data stick. "This has the information he gave me to follow him up. I convinced him to trust me, so hopefully he's receptive to you. It also has all the data that explicitly identifies all the ship ID's that are associated with our suspicious cases. That's the best I've got for you, though. If this doesn't pan out we'll need to go a different way."`
```

Reviewer: ```suggestion
			`	She rummages around and then presents you with a data stick. "This has the information he gave me to follow him up. I convinced him to trust me, so hopefully he's receptive to you. It also has all the data that explicitly identifies all the ship IDs that are associated with our suspicious cases. That's the best I've got for you, though. If this doesn't pan out we'll need to go a different way."`
```

		conversation

> 			`At length you return again to tell Giti about the system with the unstable wormhole in it. You aren't required to land elsewhere on the planet this time so it's a shorter, if equally crowded, journey to her office.`
> 			`	After the both of you spend some time pouring over things for a while she concludes that they must have some sort of special feature they're using to access it, as it makes too much sense for them to be using it for any other explanation to seem logical. "It just doesn't make sense that there'd be a system there, that they have coordinates to, that they're not using. Especially when we can't find them anywhere else."`

Reviewer: ```suggestion
			`At length you return again to tell Giti about the system with the unstable wormhole in it. You aren't required to land elsewhere on the planet this time, so it's a shorter, if equally crowded, journey to her office.`
			`	After the both of you spend some time pouring over things for a while, she concludes that they must have some sort of special feature they're using to access it. "It just doesn't make sense that there'd be a system there, that they have coordinates to, that they're not using. Especially when we can't find them anywhere else."`
```
I can't tell what is going on in the second part.

Author: That bit of the text is redundant anyway.

		not "event: knows about keystones and wormholes"
	on offer
		conversation
			`At length you return again to tell Giti about the system with the unstable wormhole in it. You aren't required to land elsewhere on the planet this time, so it's a shorter, if equally crowded, journey to her office.`
			`	After the both of you spend some time pouring over things for a while, she concludes that they must have some sort of special feature they're using to access it. "It just doesn't make sense that there'd be a system there, that they have coordinates to, that they're not using. Especially when we can't find them anywhere else."`
			choice
				`	"Do you have any other leads I can follow?"`
				`	"Even if I've found them, I don't know what they're doing or how. Any ideas?"`
			`	She looks at the many tabs displaying the information you've brought or she's found. "You remember that online community on unexplained kidnappings? I came across a man who might, just maybe, know something useful. He claims to have found something odd related to cases in the set in your data, and one of the cases in your set is the disappearance of his wife, which is where his motivation comes from.`

> 			`	"His theory is... well it's a bit crackpot for the local authorities. I wouldn't have believed it either were it not for the vastly superior swathe of information we now have at our disposal, but it might be what you need. I think you should go seek him out on Tarazed."`

Author: I changed it to `<planet>`

> 			`	"His theory is... well it's a bit crackpot for the local authorities. I wouldn't have believed it either were it not for the vastly superior swathe of information we now have at our disposal, but it might be what you need. I think you should go seek him out on <planet>."`

Reviewer: ```suggestion
			`	"His theory is... well, it's a bit crackpot for the local authorities. I wouldn't have believed it either, were it not for the vastly superior swathe of information we now have at our disposal, but it might be what you need. I think you should go seek him out on <planet>."`
```

			`	You clarify that technically it's a lead on cases that match the same profile as his wife's, not strictly his wife's case. His expression falters momentarily and then he seems to find some internal resolve to draw upon. "Good enough for me... Thank you, for your honesty."`
			label lab

> 			`	After a short return trip, you reach what looks like a small private spaceport. From the extra equipment and unusual ship designs, you realize it must be one of Tarazed's research shipyards. Goraz's laboratory is an entire docking bay with a modified Blackbird. Despite its size, it is conspicuously absent any other personnel. He takes you on board and, noting your attention to your surroundings, explains that he has few assistants that are typically off-world most of the time as he walks you to the communications room.`

Reviewer: ```suggestion
			`	After a short return trip, you reach what looks like a small private spaceport. From the extra equipment and unusual ship designs, you realize it must be one of Tarazed's research shipyards. Goraz's laboratory is an entire docking bay with a modified Blackbird, but despite its size, it is conspicuously absent any other personnel. He takes you on board and leads you to the communications room; noting your attention to your surroundings, Goraz explains that he has a few assistants, but they are off-world most of the time.`
```
This is still a little clunky I think.

Author: I made lots of changes to that section, and I think I incorporated everything you suggested.

			`	Goraz says, "Here, let me take a look at what you've brought me." He's eyeing your pockets.`
			choice
				`	"It's all in this data stick."`
				`	"I think this is what you've been looking for."`
			`	Goraz takes the data stick you brought and plugs it into the communicator. After several minutes of mucking about with list formats and muttering incoherently to himself he pauses as a search result collates matches on his screen. You're not sure exactly what you're looking at, but when it finishes he lets out a long slow breath and comments to himself in a low voice. "Bastijn Biesterveld, you absolute legend. He was right - it's the interference."`
			`	He turns back to you and explains. "Normally interference isn't something that's logged, but when you're in civilized space with hyperspace relays, your ship does a final handshake with the local relay station as it jumps." He pauses briefly, maybe to see if you understand him.`
			`	You see a long technical explanation coming on.`
			choice
				`	"Skip the technical bits, please. Can you tell me what I need to do?"`
					goto skip
				`	"I've had helped with hyperspace relay issues recently, and I'm excited to learn more."`

> 			`	He continues, "Right. So, the local relay station your ship contacts sends that handshake signature to the destination station so it knows exactly who you are to be able to reconnect communications at the other end. It's a fully automatic process, and it's independent of ship IDs. It's just buried in there as a check from back in the days when hyperdrives would occasionally explode and we'd need to work out who went missing. It gets logged and automatically overwritten every couple days, and someone would probably have gotten rid of it if it wasn't so deeply baked into the way the system works. If you know when a ship jumped and where it jumped to, you can use it to confirm it didn't blow up; and if there's something wrong with the communications equipment you can use the log info to identify if a specific ship is the same ship talking to you later, but neither of those are particularly useful since the same information can be acquired more easily and in much higher quality through other means."`

Author: The ridiculously long response is to warn the player that they are about to receive an information dump that would not be interesting to anyone except, perhaps, the person who put it in the mission txt in the first place.

> 			`	He continues, "Right. So, the local relay station your ship contacts sends that handshake signature to the destination station so it knows exactly who you are to be able to reconnect communications at the other end. It's a fully automatic process, and it's independent of ship IDs. It's just buried in there as a check from back in the days when hyperdrives would occasionally explode and we'd need to work out who went missing. It gets logged and automatically overwritten every couple days, and someone would probably have gotten rid of it if it wasn't so deeply baked into the way the system works. If you know when a ship jumped and where it jumped to, you can use it to confirm it didn't blow up; and if there's something wrong with the communications equipment you can use the log info to identify if a specific ship is the same ship talking to you later, but neither of those are particularly useful since the same information can be acquired more easily and in much higher quality through other means."`

Author: I was scolded in another PR for removing an overly wordy text dump filled with network jargon. The "I'm excited to learn more" is supposed to be dripping with sarcasm, and the "He continues, 'Right. So the" emphasizes that a rambling block of technobabble nonsense is to follow. It lets the player know they can stop reading and skip to the end.

> 			`	He continues, "Right. So, the local relay station your ship contacts sends that handshake signature to the destination station so it knows exactly who you are to be able to reconnect communications at the other end. It's a fully automatic process, and it's independent of ship IDs. It's just buried in there as a check from back in the days when hyperdrives would occasionally explode and we'd need to work out who went missing. It gets logged and automatically overwritten every couple days, and someone would probably have gotten rid of it if it wasn't so deeply baked into the way the system works. If you know when a ship jumped and where it jumped to, you can use it to confirm it didn't blow up; and if there's something wrong with the communications equipment you can use the log info to identify if a specific ship is the same ship talking to you later, but neither of those are particularly useful since the same information can be acquired more easily and in much higher quality through other means."`

Author: The dripping sarcasm is more obvious now, and I've modified the text, so he is giving what appears to be a serious lecture about network issues.

			label lead
			`	You clarify that technically it's a lead on cases that match the same profile as his wife's, not strictly his wife's case. His expression falters momentarily and then he seems to find some internal resolve to draw upon. "Good enough for me... Thank you, for your honesty."`
			label lab

> 			`	After a short return trip, you reach what looks like a small private spaceport. From the extra equipment and unusual ship designs, you realize it is a Tarazed research shipyard. Goraz's laboratory is an entire docking bay with a modified Blackbird, but despite its size, it is conspicuously absent any other personnel. He takes you on board and noting your attention to your surroundings, explains that he has a few assistants, but they are off-world most of the time.`

Reviewer: ```suggestion
			`	After a short return trip, you reach what looks like a small private spaceport. From the extra equipment and unusual ship designs, it must be a Tarazed Corporation research shipyard. Goraz's laboratory is an entire docking bay with a modified Blackbird, but despite its size, it is conspicuously absent any other personnel. He takes you on board and, noting your attention to your surroundings, explains that he has a few assistants who are off-world most of the time.`
```

			label lead

> 			`	You clarify that technically it's a lead on cases that match the same profile as his wife's, not strictly his wife's case. His expression falters momentarily and then he seems to find some internal resolve to draw upon. "Good enough for me... Thank you, for your honesty."`

Reviewer: I wonder if it's possible to contact Goraz's wife on Elenchus. Maybe, depending on what actions you take, you could get her back to him yourself, or she might come back on another transport (which means he'd send you a message), or she'd refuse and stay on Elenchus.

Author: That belongs in a follow-up mission. Two human ships with jump drives are still at Elenchus (presumably with a massive rental payment) so they can take people back and forth. I expect Elenchus also has a few jump drives in reserve.

Author: We need some post-campaign "check on your friends" missions, and Goraz should certainly be one of them.

			`	Goraz says, "Here, let me take a look at what you've brought me." He's eyeing your pockets.`
			choice
				`	"It's all in this data stick."`
				`	"I think this is what you've been looking for."`
			`	Goraz takes the data stick you brought and plugs it into the communicator. After several minutes of mucking about with list formats and muttering incoherently to himself, a search result collates matches on his screen and he pauses. You're not sure exactly what you're looking at, but when it finishes he lets out a long slow breath and comments to himself in a low voice. "Bastijn Biesterveld, you absolute legend. He was right - it's the interference."`
			`	He turns back to you and explains. "Normally interference isn't something that's logged, but when you're in civilized space with hyperspace relays, your ship does a final handshake with the local relay station as it jumps." He pauses briefly, maybe to see if you understand him.`

> 			`	You see a long technical explanation coming on.`

Reviewer 2: I enjoyed how this actually came together.

> 		government "Smuggler (Hai Trafficker)"

Author: You need to use continuous weapons, and hit smugglers that are not near other ships.

Changes to governments.txt are outside the scope of this PR. That would have to be in a separate smuggler balance PR.

> 		government "Smuggler (Hai Trafficker)"

Author: You should either open an issue for this, or do a PR with balance changes.

> 		government "Smuggler (Hai Trafficker)"

Author: I think the solution is to:

1. Remove all affinities to the Smugglers from governments.txt except for pirates since nobody is supposed to be able to identify these except pirates.
2. Give all Smuggler fleets the "target" personality so they'll be labeled "mission target" and blink on the radar, just like any other mission target.

> 		government "Smuggler (Hai Trafficker)"

Author: In #7734, the Smugglers will be labeled as mission targets. It only solves half the problem; the Navy might blow them up before you get to them. You'll be able to find them first though.

	on offer
		conversation
			`As you maneuver to board this ship you send a message telling them that you know they're smugglers and demand their surrender. There is no response, but once you dock and open your side of the airlock, the door to the smuggler ship <origin> also opens partway and a Hai pokes their head through. The Hai's eyes are wide with fear but also determination as they shout at you through the docking tube.`
			choice
				`	"Come on, you can do it! Just push a little harder!"`
				`	"There should be an access panel above you! Maybe you can short-circuit the door?"`
			`	The Hai says, "This door is jammed, they're locking down the ship and going to blow it up!" In the background you hear a booming sound. "They're using the keystones, tell them they're using the keystones!"`

> 			`	Inside your ship you hear alarms blare into full volume and you have just enough time to nod an acknowledgment to the doomed Hai before the airlock's automatic safeties shut it. A moment later, you hear an explosion and the sound of tearing metal as the docking tube is shredded and your ship is forced away from the expanding debris of the smuggler ship. When you return to the controls it seems the smugglers signaled for help before the ship exploded. An escape pod is rapidly escaping and pirates are on a swift approach. Apparently the smugglers care more about not being caught than about finishing their mission.`

Reviewer: ```suggestion
			`	Inside your ship you hear alarms blare into full volume and you have just enough time to nod an acknowledgment to the doomed Hai before the airlock's automatic safeties shut it. A moment later, you hear an explosion and the sound of tearing metal as the docking tube is shredded and your ship is forced away from the expanding debris of the smuggler ship. When you return to the controls it seems the smugglers signaled for help before the ship exploded. An escape pod is flying away from the scene; apparently the smugglers care more about not being caught than about finishing their mission.`
```

Author: This removes the information that pirates are approaching.

Reviewer: The sentence before didn't make sense, and we're told pirates are coming in the next paragraph.

Author: The next paragraph doesn't say the ships are pirates, though... but I'm not sure that it needs to.

Reviewer: It was repetitive anyway.

> 			`	You need to relay this to Giti, but the keystones are a Hai thing and that message didn't really explain much; you'll have to contact Sayari or Teeneep first which means crossing the blockade again. You begin to set a course for <planet>, but your scanners report multiple new contacts; hostile ships entering the system. You prepare for battle.`

Reviewer: ```suggestion
			`	You need to relay this to Giti, but the keystones are a Hai thing and that message didn't really explain much; you'll have to contact Sayari or Teeneep first, which means crossing the blockade again. You begin to set a course for <planet>, but your scanners report multiple new contacts; hostile ships entering the system. You prepare for battle.`
```

		conversation

> 			`The progressive search of <origin> still seems to be underway, and a cursory question directed to the Navy personnel now managing the spaceport indicates that they're less than happy about it. Despite the Navy promising to leave as soon as they've searched and confirmed the status of a handful of remaining pirate gang's hideouts; those gangs remain resolutely hidden and haven't been exposed by any of their compatriots as yet.`

Reviewer: ```suggestion
			`The progressive search of <origin> still seems to be underway, and a cursory question directed to the Navy personnel now managing the spaceport indicates that they're less than happy about it. Despite the Navy promising to leave as soon as they've searched and confirmed the status of a handful of remaining pirate gang's hideouts, those gangs remain resolutely hidden and haven't been exposed by any of their compatriots as yet.`
```

> 			`	Fortunately this makes it easy to find Sayari who has comparatively little to do at the moment and is mostly just helping to keep everyone communicating effectively. When you tell her of the system you found and the message from the Hai on the smuggler ship she winces. "What a terrible fate, but it does confirm what we suspected. We'll have to see what more we can find out in the meantime while we're still bogged down here."`

Reviewer: ```suggestion
			`	Fortunately this makes it easy to find Sayari, who has comparatively little to do at the moment and is mostly just helping to keep everyone communicating effectively. When you tell her of the system you found and the message from the Hai on the smuggler ship, she winces. "What a terrible fate, but it does confirm what we suspected. We'll have to see what more we can find out in the meantime while we're still bogged down here."`
```
